gateway-ha/src/main/java/io/trino/gateway/ha/security/LbTokenUtil.java

```diff
@@ -8,7 +8,7 @@
 
 import java.security.interfaces.RSAPublicKey;
 
-public static final class LbTokenUtil
+public final class LbTokenUtil
```

Comment:
This change is unrelated to "Declare strings as static" commit. Also, I'm not sure why you removed `static`.